Dockerfile

```diff
@@ -43,12 +43,12 @@ RUN mkdir /ucentral-data
 RUN apk add --update --no-cache librdkafka mariadb-connector-c libpq unixodbc
 
 COPY --from=builder /ucentralgw/cmake-build/ucentralgw /ucentral/ucentralgw
-COPY --from=builder /cppkafka/cmake-build/src/lib/* /lib/
+COPY --from=builder /cppkafka/cmake-build/src/lib64/* /lib/
```

Comment:
Changed that to `/lib` two weeks ago since there was a change in the cmake package in Alpine Linux. Was that changed again or what is the reason for the rename?

Reply:
It failed for me, but my guess is that I could be using old Alpine image. Let me revert changes in Dockerfile so we would use changes from your image